src/core/load-fragment-gperf.gperf.in

```diff
@@ -361,6 +361,7 @@ Unit.ConditionVirtualization,                 config_parse_unit_condition_string
 Unit.ConditionHost,                           config_parse_unit_condition_string,                 CONDITION_HOST,                     offsetof(Unit, conditions)
 Unit.ConditionKernelCommandLine,              config_parse_unit_condition_string,                 CONDITION_KERNEL_COMMAND_LINE,      offsetof(Unit, conditions)
 Unit.ConditionKernelVersion,                  config_parse_unit_condition_string,                 CONDITION_KERNEL_VERSION,           offsetof(Unit, conditions)
+Unit.ConditionVersion,                        config_parse_unit_condition_string,                 CONDITION_VERSION,                  offsetof(Unit, conditions)
```

Comment:
as mentioned, please deprecate ConditionKernelVersion at the same time, i.e. keep bith entries here, but map them to the same CONDITION_VERSION enum value